From 808a9a82ce19cf8b181b701d8d2f705a69e7ed00 Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@ti.com>
Date: Mon, 10 Sep 2012 15:19:17 -0400
Subject: [PATCH 4/4] ARM: dts: AM33XX: adjust leds to use the corrected gpio
 label

With the am33xx.dtsi now using gpio labels that match the AM33xx
documentation, we need to refer to GPIO1 by the "gpio1" label.

Signed-off-by: Matt Porter <mporter@ti.com>
---
 arch/arm/boot/dts/am335x-bone.dts |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-bone.dts b/arch/arm/boot/dts/am335x-bone.dts
index f2ee748..d4a8756 100644
--- a/arch/arm/boot/dts/am335x-bone.dts
+++ b/arch/arm/boot/dts/am335x-bone.dts
@@ -41,27 +41,27 @@
 
 			led0 {
 				label = "beaglebone:green:usr0";
-				gpios = <&gpio2 21 0>;
+				gpios = <&gpio1 21 0>;
 				linux,default-trigger = "heartbeat";
 				default-state = "off";
 			};
 
 			led1 {
 				label = "beaglebone:green:usr1";
-				gpios = <&gpio2 22 0>;
+				gpios = <&gpio1 22 0>;
 				linux,default-trigger = "mmc0";
 				default-state = "off";
 			};
 
 			led2 {
 				label = "beaglebone:green:usr2";
-				gpios = <&gpio2 23 0>;
+				gpios = <&gpio1 23 0>;
 				default-state = "off";
 			};
 
 			led3 {
 				label = "beaglebone:green:usr3";
-				gpios = <&gpio2 24 0>;
+				gpios = <&gpio1 24 0>;
 				default-state = "off";
 			};
 		};
-- 
1.7.10.4

